che--typescript/build.sh

```diff
@@ -29,17 +29,29 @@ generate_dto() {
       echo "Snapshot version is used in pom.xml. Generating again pom.xml";
     fi
   fi
-
-  DTO_CONTENT=$(cd $DIR && docker run -i --rm -v "$HOME/.m2:/root/.m2" -v "$PWD"/dto-pom.xml:/usr/src/mymaven/pom.xml -w /usr/src/mymaven maven:3.3-jdk-8 /bin/bash -c "mvn -q -DskipTests=true -Dfindbugs.skip=true -Dskip-validate-sources install  && cat target/dto-typescript.ts")
+  
+  docker rm -f che-typescript-build 2> /dev/null
+  docker run -d --name che-typescript-build -v "$HOME/.m2:/root/.m2" -w /usr/src/mymaven maven:3.3-jdk-8 tail -f /dev/null
+  docker cp $DIR/dto-pom.xml che-typescript-build:/usr/src/mymaven/pom.xml  
+  cd $DIR && docker exec -ti che-typescript-build /bin/bash -c "cd /usr/src/mymaven && mvn -q -DskipTests=true -Dfindbugs.skip=true -Dskip-validate-sources clean install && ls target/"
+  docker cp che-typescript-build:/usr/src/mymaven/target/dto-typescript.ts /tmp/dto-typescript.ts
+  DTO_CONTENT=$(cat /dto-typescript.ts)
+  docker rm -f che-typescript-build 2> /dev/null
 
   # Check if maven command has worked or not
   if [ $? -eq 0 ]; then
     # Create directory if it doesn't exist
     if [ ! -d "${DIR}/src/api/dto" ]; then
       mkdir ${DIR}/src/api/dto
     fi
+    
+    if [ ! -s /tmp/dto-typescript.ts ]; then
```

Comment:
If the build fails `docker run -d --name che-typescript-build -v "$HOME/.m2:/root/.m2" -w /usr/src/mymaven maven:3.3-jdk-8 tail -f /dev/null` the `/tmp/dto-typescript.ts` will not be copied over from via `docker cp che-typescript-build:/usr/src/mymaven/target/dto-typescript.ts /tmp/dto-typescript.ts`. This checks for it, provides error message, and exits the build.